pkgs/by-name/ep/epson_201310w/package.nix

```diff
@@ -0,0 +1,68 @@
+{
+  lib,
+  stdenv,
+  fetchurl,
+  rpmextract,
+  autoreconfHook,
+  file,
+  libjpeg,
+  cups,
+}:
+stdenv.mkDerivation (finalAttrs: {
+  pname = "epson_201310w";
+  version = "1.0.1";
+
+  src = fetchurl {
+    url = "https://download3.ebz.epson.net/dsc/f/03/00/15/64/77/6f825c323a9f4b8429e3f9f0f4e9c50b1b15f583/epson-inkjet-printer-201310w-${finalAttrs.version}-1.src.rpm";
+    hash = "sha256-596qMon5KmMHe7mh+ykdpWeazJOCGA6HmmtvdGheRmA=";
+  };
+
+  nativeBuildInputs = [
+    rpmextract
+    autoreconfHook
+    file
+  ];
+  buildInputs = [
+    libjpeg
+    cups
+  ];
+
+  unpackPhase = ''
+    rpmextract $src
+    tar -zxf epson-inkjet-printer-201310w-1.0.1.tar.gz
+    tar -zxf epson-inkjet-printer-filter-1.0.2.tar.gz
+    substituteInPlace epson-inkjet-printer-201310w-1.0.1/ppds/EPSON_L120.ppd --replace-fail "/home/epson/projects/PrinterDriver/P2/_rpmbuild/SOURCES/epson-inkjet-printer-201310w-1.0.1/watermark" "$out/watermark"
+    cd epson-inkjet-printer-filter-1.0.2
+  '';
+
+  postInstall = ''
```

Comment:
`so`- and `ppd`-files end up with executable bit set, that's probably not intended.  That probably coming from `cp -a` or from `install`.

Maybe directories can be copied with `cp -r` and single files with `install --mode=0644` (just a suggestion, others might prefer something else).  Only the filter `epson_inkjet_printer_filter` should end up with executable bit set.